lib/turn.rb

```diff
@@ -0,0 +1,27 @@
+require 'pry'
+require './lib/card'
+
+class Turn
+  atttr_reader :guess, :card, :answer
+
+  def initalize(guess, card, answer)
+    @guess = guess
+    @card = card
+    @answer = answer
+
+  print @card
+
+  @user_answer = user_answer
+  user_answer = gets
+
+    @user_answer.downcase == @answer.downcase
+
+
+      if true print @user_answer.chop + " is Correct!"
+
+      else print "Incorrect"
+
+
+  end
```

Comment:
Got a bit ahead of yourself in regards to the functionality of this project -- Follow the interaction pattern to get your methods/logic set up, and then Iteration 3 was to display questions and get user input. Make sure you're reading interaction patterns, and going in order of what we recommend (outlined by iterations).